frontend/components/main.jsx

```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { PEOPLE } from '../helpers';
-import { xsSpacing } from '../constants';
+import { sidecar, tamarind, juniper, dustStorm, whiteLilac } from '../constants';
```

Comment:
doesn't look like these are all used?

Reply:
Yeah I was playing around with the colors, I'll remove the ones I don't end up using